uitools/unit_tests/BasemapGalleryItemUnitTest/tst_basemapgalleryitemunittest.cpp

```diff
@@ -19,33 +19,30 @@
 #include "AutoDisconnector.h"
 #include "Basemap.h"
 #include "BasemapGalleryItem.h"
+#include "Error.h"
```

Comment:
It looks like unit tests were not updated for long time. Fixed. Maybe we should add them in GitHub action.

Reply:
I like this idea.